_layouts/blank.html

```diff
@@ -6,7 +6,7 @@
     {% include svg-icons.html %}
     <script>
       if ('serviceWorker' in navigator) {
-        navigator.serviceWorker.register('{{ site.url }}/sw.js');
+        navigator.serviceWorker.register('/sw.js');
```

Comment:
Can we also add a `catch()` just in case the service worker fails to register? 😬 

Something like:
```js
navigator.serviceWorker
  .register()
  .catch(error => {
    console.error('Service worker registration failed:', error);
  });
```